src/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -721,33 +721,50 @@ public String getID() {
     // get routines to access context info fields
     //
     // ---------------------------------------------------------------------------------------------------------
+    /** shortcut to {@link CommonInfo#getName()} */
```

Comment:
I would rather have the javadoc in VariantContext.java instead of CommonInfo.  Or both.